experiments/utils.py

```diff
@@ -123,17 +123,21 @@ def enroll(self, experiment_name, alternatives, force_alternative=None):
 
         return chosen_alternative
 
-    def get_alternative(self, experiment_name):
+    def get_alternative(self, experiment_name, request=None):
         """
         Get the alternative this user is enrolled in.
         """
+        disabled = False
+        if request and hasattr(request, 'experiments'):
+            disabled = experiment_name in request.experiments.disabled_experiments
```

Comment:
by reading this it's very difficult to understand the issue, trying to guess... so the someone is using the site when another person disable an experiment or himself/herself disables it and then it wasn't being tracked?

Reply:
If an experiment is disabled for the current request (by a "conditional"), then we must display the "control" alternative. That is that this change is about. You comment could map onto another change, though.

Should I change something?

Reply:
No, just a general question. If I don't understand what's going on, how can I help you get this out of the door?

Reply:
sure thing